frontend/packages/operator-lifecycle-manager-v1/locales/en/olm-v1.json

```
@@ -1,4 +1,8 @@
 {
   "Ecosystem": "Ecosystem",
-  "Installed Extensions": "Installed Extensions"
+  "Extension Catalog": "Extension Catalog",
+  "Installed Extensions": "Installed Extensions",
+  "Extension Catalog items": "Extension Catalog items",
+  "No Extension Catalog items found": "No Extension Catalog items found",
+  "Please check that OLM v1 is configured and that you have created at least one valid ClusterCatalog.": "Please check that OLM v1 is configured and that you have created at least one valid ClusterCatalog."
```

I know the console uses "please" in some messages, but thought it might be worth noting that we avoid it in the documentation.
```suggestion
  "Check that OLM v1 is configured and that you have created at least one valid ClusterCatalog.": "Check that OLM v1 is configured and that you have created at least one valid ClusterCatalog."
```

---

Thanks, I'll update that.